src/kit.ts

```diff
@@ -539,7 +540,8 @@ async function collectDevBatVars(devbat: string, args: string[], major_version:
   try {
     await fs.unlink(envpath);
   } catch (error) {}
-  await fs.writeFile(batpath, bat.join('\r\n'));
+  const batContent = bat.join('\r\n');
```

Comment:
Minor: batContent can be set one line earlier, then fs.writeFile can use batContent as second parameter.

Reply:
Don't understand, batContent  used latter

Reply:
I don't see it now but I thought I saw twice the following: bat.join('\r\n').